crates/cxx-qt-build/src/cfg_evaluator.rs

```diff
@@ -0,0 +1,167 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+// SPDX-FileContributor: David Tolnay <dtolnay@gmail.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+// Modified from
+// https://github.com/dtolnay/cxx/blob/2193addc308c99512b8757431abfaf87cc54c086/gen/build/src/cargo.rs
```

Comment:
Do we want to include the original licenses from the CXX repository here?

I know CXX is also MIT & Apache dual-licensed, but maybe we can at least link to them to ensure we're doing things correctly :thinking:

Reply:
We wouldn't need to link to them as they are already in the LICENSES folder and specified by the SPDX identifier. However the `SPDX-FileCopyrightText` could instead be CXX authors or a name they are using as it's primarily come from that source.

Reply:
Updated to "CXX Authors" with the file contributors, so should be correct now.